lib/libc/newlib/libc-hooks.c

```diff
@@ -174,33 +177,33 @@ int _read(int fd, char *buf, int nbytes)
 
 	return z_impl_zephyr_read_stdin(buf, nbytes);
 }
-__weak FUNC_ALIAS(_read, read, int);
```

Comment:
What intent of this change? without weak alias nobody can override syscals

Reply:
OMG, thanks for pointing me on that. This went errornously within the commit - the `__weak` is back now.